chain/client/src/metrics.rs

```diff
@@ -189,6 +189,10 @@ pub(crate) static TRANSACTION_RECEIVED_VALIDATOR: Lazy<IntGauge> = Lazy::new(||
     try_create_int_gauge("near_transaction_received_validator", "Validator received a transaction")
```

Comment:
The description is no longer accurate. Should be ‘Validator received and recorded a transaction’